### ISSUE TITLE
logging breaks vanilla html

### ISSUE THREAD

**Issue by JuanICasareski** [wallabyjs/console-ninja#65]
Love the extension 💘.
It works great with the preact part of the project. But when I try to place a `console.log(...)` on vanilla html

I get the following on the vite hmr overlay (preact template),
I assume the unwanted code come from the wrapper for changing the default behaviour of `console.log(...)`

```
[plugin:prefresh] unknown: Unexpected token (10:40)
13 |         const onExi/* eslint-disable */;(oo_oo(),t = () => { console., `1f5dc3bb_3`l)og("onExit") };

C:/Users/Juani/route/to/index.html
8  |          const healthcareServices = [];
9  |  
10 |          const onOk =/* eslint-disable */;(oo_oo(), () => { console.l, `1f5dc3bb_0`o)g("onOk") };
   |                                          ^
11 |          const onError = (er/* eslint-disable */;(oo_oo(),ror) => { console.log("onErr, `1f5dc3bb_1`o)r", error) }; 
12 |          const onSubmit/* eslint-disable */;(oo_oo(), = () => { console.log, `1f5dc3bb_2`()"onSubmit") };
```

Here's the code that that triggers it

```
const onOk = () => { console.log("onOk") };
const onError = (error) => { console.log("onError", error) }; 
const onSubmit = () => { console.log("onSubmit") };
const onExit = () => { console.log("onExit") }; 
```

**Comment by smcenlly** [comment]
We couldn't work out how to reproduce your problem.

Can you please provide us with step-by-step details for reproducing the error, or else provide us with a sample repo that has the same problem?

**Comment by Zubbbz** [comment]
I also have this issue [see here](https://youtu.be/VdKJRQa9OZs)

**Comment by ArtemGovorov** [comment]
@Zubbbz Thanks for recording the video. Can you please:
- update to the latest version of the extension and try to see if the issue is still there?
- if the issue can still be reproduced, can you please paste the full content of the `index.html` file from your video?

@JuanICasareski What OS version are you using?

**Comment by NikGovorov** [comment]
Thanks for reporting the issue. It is fixed and the fix is published in v0.0.69.
